elki-core-math/src/main/java/elki/math/statistics/dependence/mcde/MCDETest.java

```diff
@@ -48,19 +48,25 @@
      */
     public int[] index;
 
+    /**
+     * Length of data
+     */
+    public int len;
+
     /**
      * Constructor.
      *
      * @param idx Array containing index (position) values
      */
-    public RankStruct(int[] idx) {
+    public RankStruct(int[] idx, int len) {
```

Comment:
Will the length ever be anything other than idx.length?

Reply:
Done